cli/pkg/app/app.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"io"
+	"os"
+)
+
+const api string = "http://localhost:8000"
+
+type Stream struct {
+	Out io.Writer
+	Err io.Writer
+}
+
+//func FetchHubResource(cli CLI, api string) ([]byte, bool, error) {
```

Comment:
should probably be deleted :stuck_out_tongue_closed_eyes:

Reply:
Yeah :smile:  today Sunil restructured the code a bit so code is gonna change now :sweat_smile: